deploy/helm/sumologic/README.md

```diff
@@ -24,8 +24,8 @@ Parameter | Description | Default
 `sumologic.endpoint` | Sumo API endpoint; Leave blank for automatic endpoint discovery and redirection. | `Nil`
 `sumologic.collectorName` | The name of the Sumo Logic collector that will be created in the SetUp job.  Defaults to `clusterName` if not specified. | `Nil`
 `sumologic.clusterName` | An identifier for the Kubernetes cluster. | `kubernetes`
-`sumologic.httpProxy` | HTTP proxy URL | ``
-`sumologic.httpsProxy` | HTTPS proxy URL | ``
+`sumologic.httpProxy` | HTTP proxy URL | `Nil`
```

Comment:
Why `Nil`? Isn't an empty string a better value for this? i.e. `""`?

Reply:
We are using `Nil` as empty string across all file